zap/src/main/java/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -1294,6 +1307,32 @@ public void process(RecordHistory recordHistory) {
             result =
                     new ApiResponseElement(
                             name, getAlertParam(ApiException.Type.BAD_VIEW).getOverridesFilename());
+        } else if (VIEW_LOGGING_GENERAL.equals(name)) {
```

Comment:
I don't see the need for two, if the user just wants these they can filter.

Reply:
Okay